src/legacy/core_plugins/vega/public/_vega_vis.scss

```diff
@@ -118,12 +122,12 @@
 
   td.key {
     color: $euiColorMediumShade;
-    max-width: $euiSize * 10;
+    max-width: $vegaTooltipMaxWidth * 0.4;
```

Comment:
These calculations are interesting, however since you're now calculating a percentage, I think it's best to actually add the column width directly to the table column as a percentage. Like `<td width="40%">` That way if we need to readjust the sizing of the tooltip the table should auto-adjust.

Reply:
I've tried it with width percentages on the `<td>`  elements but it required using a fixed table layout and setting the table width to 100%, like this:
```
.vgaVis__tooltip table {
    table-layout: fixed;
    width: 100%;
}
```
The width is needed in order to overflow the cells.  The problem is the table ends up being full width even if it doesn't need to be? 

With 100% width it looks fine for a wide tooltip:
![image](https://user-images.githubusercontent.com/7903188/54652252-9c51e380-4a8c-11e9-9e6c-2ef0f615ec34.png)

But a small tooltip ends up with a lot of whitespace:
![image](https://user-images.githubusercontent.com/7903188/54652383-16826800-4a8d-11e9-8ec9-7769ba016e09.png)

Reply:
Ah ok, thanks for investigating that.